sdk/containerregistry/Azure.Containers.ContainerRegistry/README.md

```diff
@@ -126,11 +126,12 @@ Uri endpoint = new Uri(Environment.GetEnvironmentVariable("REGISTRY_ENDPOINT"));
 
 // Create an invalid ContainerRepositoryClient
 string fakeRepositoryName = "doesnotexist";
-ContainerRepositoryClient client = new ContainerRepositoryClient(endpoint, fakeRepositoryName, new DefaultAzureCredential());
+ContainerRegistryClient client = new ContainerRegistryClient(endpoint, new DefaultAzureCredential());
+ContainerRepository repository = client.GetRepository(fakeRepositoryName);
 
 try
 {
-    client.GetProperties();
+    repository.GetProperties();
```

Comment:
Keeping in mind this is the first file in the PR, is `ContainerRepository` a client? I'm assuming not, since it doesn't end in `Client` like even Storage sub-clients do. If that's the case, does `GetProperties()` really need to be a method then? Is it a separate fetch, or can it just be a property of `repository` if, in fact, `repository` actually has the data?

Reply:
`GetProperties()` calls the service to retrieve the properties of a repository. @KrzysztofCwalina had suggested this approach to the API to follow what the Cosmos API is doing with helper classes like [CosmosDatabase](https://apiview.dev/Assemblies/Review/0de57f00b1ad4fdab2fa218f59863c78#Azure.Cosmos.CosmosDatabase) and [CosmosContainer](https://apiview.dev/Assemblies/Review/0de57f00b1ad4fdab2fa218f59863c78#Azure.Cosmos.CosmosContainer).  

In ACR, `ContainerRepository` and `RegistryArtifact` are helper classes like these that organize operations around a given resource.  I recognize that this is nonstandard, but we believe it simplifies the API.